installer/PowerToysSetup/Product.wxs

```diff
@@ -3,6 +3,8 @@
      xmlns:util="http://schemas.microsoft.com/wix/UtilExtension"
      xmlns:netfx="http://schemas.microsoft.com/wix/NetFxExtension" >
 
+  <?define ShortcutGuideProjectName="shortcut_guide"?>
```

Comment:
I would like if we can move away from using the underscore for the modules project names and binaries.
Can we start using `ShortcutGuide`?
In the VS solution as well.

Reply:
Done